vector/src/main/scala/geotrellis/vector/convexhull/ConvexHullMethods.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2014 Azavea.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.vector
+package convexhull
+
+trait ConvexHullMethods[G <: Geometry] extends MethodExtensions[G] {
+  def convexHull: Geometry = Geometry(self.jtsGeom.convexHull)
```

Comment:
What are the potential return values of this? We should bound it to a result if we can. Seems like for point or line it doesn't make sense, and would be an empty geom, and for others it might be empty or would be a polygon.